### ISSUE TITLE
Log on Reset feature / Debug wake reset

### ISSUE THREAD

**Issue by ZavenArra** [rrivirr/rriv#90]

**Comment by ZavenArra** [comment]
@kenchong48 Did we determine the location of this hang, or can you provide me with a hardware configuration that replicates this ?

**Comment by kenchong48** [comment]
@ZavenArra No, haven't had time to explore this.  To replicate I think you can just log data periodically.  My current solution is to reset upon reaching a certain file size.  So, trying to log past that file size should cause the issue to arise.  I set it to reset if it sees a file size over 100000 bytes

**Comment by ZavenArra** [comment]
We think that when the file gets too big it might be resetting.
We also think that if a new file needs to be created, it does not log to it on that cycle.
A side effect of this is that we end up missing the logging cycle that would otherwise have occurred.

**Comment by kenchong48** [comment]
Happening on at least my branch, might be an issue I've introduced though:
https://github.com/kenchong48/rriv/tree/methaneUpdate
